src/libraries/System.Private.CoreLib/src/System/IO/Path.Windows.cs

```diff
@@ -27,6 +27,33 @@ public static partial class Path
             (char)31
         };
 
+        //Checks if the given path is available for use.
+        public static bool Exists([NotNullWhen(true)] string? path)
+        {
+            try
+            {
+                if (path == null)
+                {
+                    return false;
+                }
+
+                if (path.Length == 0)
+                {
+                    return false;
+                }
+
+                string fullPath = GetFullPath(path);
+                Interop.Kernel32.WIN32_FILE_ATTRIBUTE_DATA data = default;
+                return FileSystem.FillAttributeInfo(fullPath, ref data, returnErrorOnNotFound: true) == 0;
```

Comment:
The existing code, has another check
https://github.com/dotnet/runtime/blob/6dfbc6e2277cd8097b33e90002844364a147e74c/src/libraries/Common/src/System/IO/FileSystem.Attributes.Windows.cs#L39
Perhaps there are circumstances where GetFileAttributesEx succeeds on a nonexistent file, and one must make this check?

It is probably better to write it exactly like existing code does, passing in false for the last parameter
https://github.com/dotnet/runtime/blob/6dfbc6e2277cd8097b33e90002844364a147e74c/src/libraries/Common/src/System/IO/FileSystem.Attributes.Windows.cs#L23-L28

Reply:
Fixed